src/porepy/numerics/ad/operators.py

```diff
@@ -742,207 +816,31 @@ def _evaluate(
             depends on the operator.
 
         """
-        # Get the mixed-dimensional grid used for the dof-manager.
```

Comment:
These internal data structurs are not required anymore with `Variable.parse` being implemented